### PR TITLE
Verbose GC CRIU Support

### PR DESCRIPTION
**Pull request by RSalman**
Integrate CRIU support with new (OMR) Verbose GC APIs, invoke Verbose Manager to `prepareForCheckpoint` (close verbose logs) and `reinitializeForRestore` (reopen logs). For background, see https://github.com/eclipse/omr/pull/6850

Depends on: https://github.com/eclipse/omr/pull/6850
Fixes: https://github.com/eclipse-openj9/openj9/issues/15858

Signed-off-by: Salman Rana <salman.rana@ibm.com>

### CODE REVIEW

**Comment by RSalman** [comment]
@amicic @dmitripivkine please have a look

File: runtime/gc_base/modronapi.cpp [hunk 1]
```diff
@@ -1098,21 +1099,36 @@ j9gc_prepare_for_checkpoint(J9VMThread *vmThread)
...
 BOOLEAN
 j9gc_reinitialize_for_restore(J9VMThread *vmThread)
 {
 	MM_EnvironmentBase *env = MM_EnvironmentBase::getEnvironment(vmThread->omrVMThread);
 	MM_GCExtensionsBase *extensions = env->getExtensions();
+	MM_VerboseManagerBase *verboseGCManager = extensions->verboseGCManager;
+
+	bool result;
+
+	result = extensions->configuration->reinitializeGCThreadCountForRestore(env);
```
Comment (amicic) [inline runtime/gc_base/modronapi.cpp:1124]: merge declaration and assignment into the same line

**Comment by amicic** [comment]
update copyright year

**Comment by amicic** [comment]
Jenkins test sanity win jdk19

**Comment by amicic** [comment]
Jenkins compile aix jdk11

**Review by amicic — APPROVED** [review]

**Pull request merged**